wdl/TinyResolve.wdl

```diff
@@ -145,3 +164,56 @@ task ResolveManta {
     maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
   }
 }
+
+task ConcatTars {
+  input {
+    Array[File] manta_tloc_tars
+    Array[File] manta_unresolved_tars
+    String linux_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+    cpu_cores: 1,
+    mem_gb: 1.0,
+    disk_gb: ceil(3 * (size(manta_tloc_tars, "GB") + size(manta_unresolved_tars, "GB"))),
```

Comment:
```suggestion
    disk_gb: ceil(10 + 3 * (size(manta_tloc_tars, "GB") + size(manta_unresolved_tars, "GB"))),
```
Just to be safe add some extra disk. Disk speed also scales with size, so if it's very small it may be very slow as well.